core/storage/question/gae_models.py

```diff
@@ -125,6 +128,7 @@ def create(
             language_code: str. The ISO 639-1 code for the language this
                 question is written in.
             version: str. The version of the question.
+            linked_skill_ids: set(str). The skill ids linked to the question.
```

Comment:
Suggest passing in list(str) and keeping it as list everywhere.

In the domain layer, do set deduplication operations internally to functions and don't pass sets around. (Or, use sets everywhere -- but in that case, convert to a set immediately on retrieval and keep it as a set always throughout the whole domain + controller layer -- that way, it's really a set but we just make it a list for storage purposes only. We should go with one or the other convention, but not a hybrid.)

Reply:
Just want to confirm, you are okay with me converting to a set and then list for deduplication right?